pybaseball/split_stats.py

```diff
@@ -0,0 +1,125 @@
+import requests
+from requests.adapters import HTTPAdapter
+from requests.packages.urllib3.util.retry import Retry
+import bs4 as bs
+import pandas as pd
+import re
+
+
+def download_url(url):
+    """
+    Gets the content from the url specified
+    """
+    session = requests.Session()
+    retry = Retry(connect=3, backoff_factor=0.5)
+    adapter = HTTPAdapter(max_retries=retry)
+    session.mount('http://', adapter)
+    session.mount('https://', adapter)
+    resp = session.get(url)
+    return resp.content
+
+
+def get_split_soup(playerid, year=None, pitching_splits=False):
```

Comment:
Can you please put type annotations on the functions?

Reply:
done